pkg/grapicmd/internal/usecase/init_config.go

```diff
@@ -0,0 +1,34 @@
+package usecase
+
+import "bytes"
+
+type InitConfig struct {
+	Source   string
+	Revision string
+	Branch   string
+	Version  string
+	HEAD     bool
+	Package  string
+}
+
+func (c *InitConfig) BuildSpec() string {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method InitConfig.BuildSpec should have comment or be unexported